File: cpp/bmi.cpp
/*
 * bmi.cpp

 */


#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
    float m, w, c;
    m=w=c=0;
    
    cout << "Podaj masę: " << endl;
    cin >> m;
    cout << "Podaj wzrost w metrach: " << endl;
    cin >> w;
    c = m/(w*w);    
    
    cout << "BMI: " << c << endl;
    
    if (c < 18.5){
        cout << "niedowaga" << endl;}
    else if (c <= 24.9){
        cout << "norma" << endl;}
    else if (c >= 25){
        cout << "nadwaga" << endl;}
    else if (c >= 30){
        cout << "otyłość" << endl;}
        
    
    
    
    
	
	
	return 0;
}

File: Python/BMI.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-


def main(args):
    m = float(input("Podaj masę: "))
    print(m)
    w = float(input("Podaj wzrost w metrach: "))
    print(w)

    c = m / (w * w)
    print("Bmi: ", c)

    if c < 18.5:
        print("niedowaga")
    elif c <= 24.9:
        print("norma")
    elif c >= 25:
        print("nadwaga")
    elif c >= 30:
        print("otyłość")

    return 0


if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
